.gitignore

```diff
@@ -17,3 +17,6 @@ secrets.xml
 # logging
 logback.xml
 logback-test.xml
+
+# OS X
```

Comment:
I think we call it `macOS` now.
https://www.apple.com/macos/mojave/

Reply:
Corrected. Marking this one resolved too.